packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -764,5 +764,16 @@ export const mutation = mutationType({
         return true
       },
     })
+
+    t.boolean('moveToNextRelevantRun', {
+      description: 'Allow the relevant run for debugging marked as next to be considered the current relevant run',
+      resolve: async (source, args, ctx) => {
+        const runs = await ctx.relevantRuns.moveToNext()
+
+        ctx.emitter.relevantRunChange(runs)
```

Comment:
I feel like this event should be emitted from within `ctx.relevantRuns.moveToNext()`

Reply:
agreed!